.vscode/settings.json

```diff
@@ -16,5 +16,8 @@
     },
     "[html]": {
         "editor.defaultFormatter": "esbenp.prettier-vscode"
+    },
+    "[typescript]": {
+        "editor.defaultFormatter": "vscode.typescript-language-features"

```

Comment:
Is this necessary? Can you try removing this  once and seeing if the default formatter works or not.

Reply:
No it was not required. I have removed the vscode formatter. Will be publishing changes today.